scipy/spatial/_spherical_voronoi.py

```diff
@@ -121,10 +120,25 @@ class SphericalVoronoi:
     ...                    c='g')
     >>> # indicate Voronoi regions (as Euclidean polygons)
     >>> for region in sv.regions:
-    ...    random_color = colors.rgb2hex(np.random.rand(3))
-    ...    polygon = Poly3DCollection([sv.vertices[region]], alpha=1.0)
-    ...    polygon.set_color(random_color)
-    ...    ax.add_collection3d(polygon)
+    ...    region_size = len(region)
+    ...    for i in range(region_size):
+    ...        if i == (region_size - 1):
+    ...            next_index = 0
+    ...        else:
+    ...            next_index = i + 1
+    ...        start = sv.vertices[region][i]
+    ...        end = sv.vertices[region][next_index]
```

Comment:
I think you can make this more concise. How about:
```
n = len(region)
for i in range(n):
    start = sv.vertices[region][i]
    end = sv.vertices[region][(i + 1) % n]
```

Reply:
good idea, done, thanks